lib/2wp-utils.js

```diff
@@ -238,6 +240,23 @@ const donateToBridge = async (rskTxHelper, btcTxHelper, donatingBtcAddressInform
     return peginBtcTxHash;
 };
 
+const disableWhitelisting = async (rskTxHelper) => {
+    const latestActiveForkName = await getLatestActiveForkName();
+    const bridge = getBridge(rskTxHelper.getClient(), latestActiveForkName);
+
+    const addr = await rskTxHelper.importAccount(WHITELIST_CHANGE_PK)
+    expect(addr.slice(2)).to.equal(WHITELIST_CHANGE_ADDR);
+    await rskTxHelper.unlockAccount(addr);
+
+    await sendTxWithCheck(
+        rskTxHelper,
+        bridge.methods.setLockWhitelistDisableBlockDelay(1),
+        WHITELIST_CHANGE_ADDR,
+        (disableResult) =>  expect(Number(disableResult)).to.equal(1)
+    );
+    rskTxHelper.mine(1);
```

Comment:
`sendTxWithCheck` already mines 1 RSK block after sending the transaction. If I'm not mistaken you should be mining the amount of Bitcoin blocks that you passed as a parameter to `setLockWhitelistDisableBlockDelay` (in this case 1).

By the way, I'd put that value as a constant inside this method so it's clear what it means

Reply:
ok. 
so the `sendTxWithCheck` will always mine 1 block. does it make sense to extend `sendTxWithCheck` with  parameter `blocksToMine` to be able to change required blocks to mine? with value `1` as default.

Reply:
`sendTxWithCheck` will mine 1 **RSK** block by default. But the parameter received by `setLockWhitelistDisableBlockDelay` is the amount of **Bitcoin** blocks you need to mine for the whitelist disabling to have effect. Does that make sense?

Reply:
yes, i understand.
what i'm asking is, will i ever need to mine more then 1 block for whitelist disabling?
something like
```
    await sendTxWithCheck(
        rskTxHelper,
        bridge.methods.setLockWhitelistDisableBlockDelay(blocksToMine),
        WHITELIST_CHANGE_ADDR,
        (disableResult) =>  expect(Number(disableResult)).to.equal(1)
    );
    if (blocksToMine > 1 ) {
        rskTxHelper.mine(blocksToMine - 1);
    }


or add extra parameter to `sendTxWithCheck`...

Reply:
`blocksToMine` parameter is the amount of Bitcoin blocks to mine to have the whitelist disabled, it can be more than 1 yes. But `sendTxWithCheck` will mine Rootstock blocks